contracts/contracts/Subcall.sol

```diff
@@ -578,4 +586,20 @@ library Subcall {
             revert AccountsTransferError(status, string(data));
         }
     }
+
+    /**
+     * @notice Verify whether the origin transaction is signed by an authorized
+     * ROFL instance for the given application.
+     * @param appId ROFL app identifier
+     */
+    function roflEnsureAuthorizedOrigin(bytes21 appId) internal {
+        (uint64 status, bytes memory data) = subcall(
+            ROFL_IS_AUTHORIZED_ORIGIN,
+            abi.encodePacked(hex"55", appId)
```

Comment:
Is `hex"55"` some CBOR magic constant? Maybe add a comment for this similar to `consensusDelegate()` above.

Reply:
It stands for "CBOR byte-array of length 21 bytes".